Version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
 	<!-- Versioning property for builds and packages -->
 	<PropertyGroup>
-		<VersionPrefix>1.0.50</VersionPrefix>
```

Comment:
I don't think Garnet team is following Semantic Versioning, but if we do, shouldn't this be `1.1.0`